tools/genrules/main.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/Masterminds/glide/cfg"
+	publishing "github.com/kubernetes/publishing-bot/cmd/publishing-bot/config"
+)
+
+const originRepoPrefix = "git@github.com:openshift/"
```

Comment:
@deads2k any reason why we use SSH in glide.yaml? Just wonder.

Reply:
> @deads2k any reason why we use SSH in glide.yaml? Just wonder.

It's the format I'm used to and uses my local keys for authentication.